app/views/lobby/bits.scala

```diff
@@ -160,3 +96,36 @@ object bits:
           if e.isNow then trans.site.eventInProgress() else momentFromNow(e.startsAt)
       )
     )
+
+  def recentTopics(forumTopics: List[lila.forum.RecentTopic])(using Context) =
+    div(cls := "topic-list")(
+      frag(
+        forumTopics
+          .map: topic =>
+            val mostRecent = topic.posts.last
+            div(
+              span(
+                a(
+                  href  := routes.ForumCateg.show(topic.categId),
+                  title := topic.categId.value.split("-").map(_.capitalize).mkString(" "),
+                  cls   := s"categ-link"
+                )(
+                  lila.forum.ForumCateg.publicShortNames.get(topic.categId).getOrElse("TEAM")
+                ),
+                ": ",
+                a(
+                  href := routes.ForumPost.redirect(mostRecent.post.id),
+                  title := s"${topic.name}\n\n${titleNameOrAnon(mostRecent.post.userId)}:  ${mostRecent.post.text}"
+                )(topic.name)
+              ),
+              div(cls := "contributors")(
+                topic.contribs match
+                  case Nil => emptyFrag
+                  case contribs =>
+                    frag(contribs.flatMap(uid => Seq[Frag](userIdLink(uid), " · ")).dropRight(1)*),
+                span(cls := "time", momentFromNow(topic.updatedAt))
```

Comment:
uncontrolled user profile links on the homepage will quickly lead to wild spamming.

Reply:
instead of `frag(contribs.flatMap(uid => Seq[Frag](userIdLink(uid), " · ")).dropRight(1)` there's `fragList(contribs.map(userIdLink))`

Reply:
uncontrolled user profile links could be discussed on zulip